src/js/stores/MesosStateStore.js

```diff
@@ -34,6 +34,14 @@ function stopPolling() {
   }
 }
 
+function isSchedulerTask(task, schedulerTaskIds) {
```

Comment:
Can we make this method name a bit cleaner? `isSchedulerTask` sounds to me like it should return a boolean, not mutate an arg.